crates/indexer/Cargo.toml

```
@@ -16,18 +16,23 @@ categories = ["cryptography::cryptocurrencies", "web-programming"]
 [features]
 default = []
 geyser = [
+  "reqwest-client",
   "solana-program",
+  "serde_json",
```

```suggestion
  "serde_json",
  "solana-program",
```

I believe I added this, but I need to double-check if requiring `serde_json` is even necessary anymore.